osmaxx/excerptexport/models/excerpt.py

```diff
@@ -21,18 +26,35 @@ class Excerpt(models.Model):
     excerpt_type = models.CharField(max_length=40, choices=EXCERPT_TYPES, default=EXCERPT_TYPE_USER_DEFINED)
 
     COUNTRY_SIMPLIFICATION_TOLERANCE = 0.01
+    BUFFER_METERS = 200
 
     def send_to_conversion_service(self):
         from osmaxx.api_client.conversion_api_client import ConversionApiClient
         api_client = ConversionApiClient()
         bounding_geometry = self.bounding_geometry
         if self.excerpt_type == self.EXCERPT_TYPE_COUNTRY_BOUNDARY:
-            original_srid = self.bounding_geometry.srs
-            bounding_geometry = with_metric_buffer(self.bounding_geometry, buffer_meters=100, map_srid=original_srid)
-            if not isinstance(bounding_geometry, geos.MultiPolygon):
-                bounding_geometry = geos.MultiPolygon(bounding_geometry)
+            bounding_geometry = self.simplified_buffered()
         return api_client.create_boundary(bounding_geometry, name=self.name)
 
+    def simplified_buffered(self):
+        """
+        First simplifies, than buffers the bounding_geometry.
```

Comment:
Also, rephrase this so it doesn't sound like a side effect on `self.bounding_geometry`.